osu.Game.Tests/Visual/Gameplay/TestSceneHitErrorMeter.cs

```diff
@@ -235,8 +235,10 @@ private class TestDrawableRuleset : DrawableRuleset
 
             public override IEnumerable<HitObject> Objects => new[] { new HitCircle { HitWindows = HitWindows } };
 
+#pragma warning disable 67
```

Comment:
What's this about? I don't get a local warning.

Reply:
Nevermind, it's a compile error that r# isn't showing me.

Reply:
I'd prefer implementing these as
```
add => throw new NotImplementedException();
remove => throw new NotImplementedException();
```
instead.

Reply:
TIL you can do it for overriden events as well.